src/commodore/index.spec.ts

```diff
@@ -1,18 +1,49 @@
-import { loadFixture } from '../test/util';
-import { extractPackageFile } from './index';
+import { mkdir } from 'fs/promises';
+import { getFixturePath, loadFixture } from '../test/util';
+import { extractPackageFile, defaultConfig } from './index';
+import { clearCache } from './inventory';
 import { expect, describe, it } from '@jest/globals';
 
 const params1 = loadFixture('1/params.yml');
 const kube2 = loadFixture('2/kubernetes.yml');
 const invalid3 = loadFixture('3/params.yml');
+const pin4 = loadFixture('4/pins.yml');
 
-describe('manager/commodore/index', () => {
+jest.mock('renovate/dist/config/global');
+const renovate_global = require('renovate/dist/config/global');
+function mockGetGlobalConfig(fixtureId: string) {
+  renovate_global.getGlobalConfig.mockImplementation(() => {
```

Comment:
I get a type error. [This stackoverflow post](https://stackoverflow.com/questions/51495473/typescript-and-jest-avoiding-type-errors-on-mocked-functions) shows a few ways to fix this.